server/background-jobs.js

```diff
@@ -0,0 +1,31 @@
+import Promise from 'bluebird';
+
+import queries from './lib/queries';
+
+// const useCache = ['production', 'staging'].includes(process.env.NODE_ENV);
```

Comment:
comment it  out?

Reply:
Should be un-commented, yes. I will fix it.